json_handler_test.go

```diff
@@ -153,3 +154,26 @@ func TestJSONHandler_UnmarshalJSON_ParsesCustomMultiNestedFields(t *testing.T) {
 		t.Fatalf("not equal: expected %q, got %q", tm, h.Time)
 	}
 }
+
+func TestParseAsctimeFields_1(t *testing.T) {
+	raw := []byte(`{"asctime": ["12-05-05 22:11:08,506248"]}`)
```

Comment:
maybe pull these in a table-test?

Reply:
There's an issue with `asctime` since it doesn't include timezone information. For now, my implementation assumes the parsed time is in UTC, this assumption may be incorrect depending on the user application's logging configuration.

Reply:
there's a config value for this on the config struct 🙂

Reply:
I see